### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kbjude**
Develop branch for the Apple clone project which is read to be merged with all the features implemented. 

Third project ready for submission
No HTML or CSS-Lint errors left

Worked on these files and all content push to develop
Index.html
styles.css
README.md

The resources folder:
These stores all the resources like images and other files that I needed to complete the project

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review by Tolulope Olusakin(TSE)

**Comment by nikasulo** [comment]
# Great job with how far you've come @kbjude! :handshake: 
Just a few more steps and you can merge.

#### Horizontal scroll
You have a bit of horizontal scrolling going on in your live version. Likely you have some margin or padding setup that extends beyond the browser's width. Please look into this and make appropriate corrections!

#### No gradient
The title of the project is building with gradients and backgrounds and as such, one of the hard requirements is that your navbar has a gradient as its background color. Please see the requirement below
screenshot

#### Descriptive Readme
Nice job with your current readme file. Please take a look at this [example]() and improve yours accordingly! See the requirement below.
screenshot

### Summary
Your project has a lot of potentials to shine. Please look into the following things I outlined above and make appropriate corrections. When all this is done, please submit your project for another review :rocket: 

If you think I missed something, you need some help or you'd just like to chat, please reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [instagram](https://instagram.com/oluwadamilare_olusakin)

**Comment by kbjude** [comment]
Pull Request 2

**Changes Made:**
The following changes as requested by the TSE : 
- Gradient effected in the nav-bar
- The width and margins adjusted
- Readme.md modified.

**Request**
Review the changes that I was asked to make and grant a green light to go ahead and merge if the changes made to meet the expectation

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            <div class="footer-right-item"> <a href="#"> Job Opportunities </a></div> 
+            <div class="footer-right-item"> <a href="#"> Contact Us </a></div> 
+            <div class="footer-right-item"> <img src="./resources/usa_flag.png" alt="USA Flag" id ="footer-right-img"> </div> 
+            
+        </div>
+    </footer>
+    <div class="footer-bottom">
+        Copyright © 2014 Apple Inc. All rights reserved. 
+        <div id = "footer-lower"> <a href="#">Terms of Use </a> | <a href="#"> Privacy Policy</a>  </div> 
+    </div>
```
Comment (pelzolga123) [inline index.html]: This part should be inside `footer`.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires a few changes.

**Comment by pelzolga123** [comment]
## Hi Jude! ! :christmas_tree: 
### Your project looks :+1: great! Good job with descriptive README file!

Here are few things to correct before project can be approved:
* [ ] Bottom part should contains four element which are links (to another articles or #). You need to add links, right now you have just four images. ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

* [ ] Footer should include links like copyright, terms of use, and privacy policy. Loook at the comment above. ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

[OPTIONAL]
* It is a good practice to use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by kbjude** [comment]
Hi Olga,
I do not think I have understood the ask in this email that you have just
shared with me. I already have these links in the screenshot. Are there
links that I am missing that I am not aware of. Kindly advise.

Thank you
Jude

**Comment by pelzolga123** [comment]
Ok, the thing is, right now you have four images stored in four `div` tags inside your `footer-up` div. all you need to do is to add `a` tags to each div, so that your images can be links. You can also inspect original website structure with developer tools and see how it is done there.

**Comment by kbjude** [comment]
All changes made as asked by the reviewer:
- added the main tag
- changed the images to links
- the links below the image reviewed and confirmed availability
All set to be merged

**Review by nikasulo — APPROVED** [review]

**Comment by nikasulo** [comment]
# Really beautiful job @kbjude!
You have done a great job incorporating the requested changes, and as such, please merge this and carry on with your level of attention to detail. See you at the very top :rocket:

**Pull request merged**